src/system/MailerModule/Resources/config/services.xml

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" ?>
+
+<container xmlns="http://symfony.com/schema/dic/services"
+           xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+           xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
+
+    <parameters>
+        <parameter key="zikula_mailer_module.container.link_container.class">Zikula\MailerModule\Container\LinkContainer</parameter>
+        <parameter key="zikula_mailer_module.form.config_type.class">Zikula\MailerModule\Form\Type\ConfigType</parameter>
+        <parameter key="zikula_mailer_module.form.test_type.class">Zikula\MailerModule\Form\Type\TestType</parameter>
+        <parameter key="twig.extension.zikula_mailer_module.class">Zikula\MailerModule\Twig\TwigExtension</parameter>
+        <parameter key="zikula_mailer_module.api.mailer.class">Zikula\MailerModule\Api\MailerApi</parameter>
+    </parameters>
+
+    <services>
+        <service id="zikula_mailer_module.container.link_container" class="%zikula_mailer_module.container.link_container.class%">
+            <argument type="service" id="translator.default" />
+            <argument type="service" id="jms_i18n_routing.router" />
+            <argument type="service" id="zikula_permissions_module.api.permission" />
+            <tag name="zikula.link_container"/>
+        </service>
+
+        <service id="zikula_mailer_module.form.config_type" class="%zikula_mailer_module.form.config_type.class%">
+            <argument type="service" id="translator.default" />
+            <argument type="service" id="zikula.dynamic_config_dumper" />
```

Comment:
`<argument>%swiftmailer%</argument>` I think will get you want you need, unless you need the dumper.